### PR TITLE
Feat/RTENU-59 basic stylings and router

### PR DESCRIPTION
**Pull request by NinaDang97**
**Done:**
- [x] Use custom font system (Exo2)
- [x] Use MUI with Emotions for styling
- [x] Two complete views (Access Denied and Not Found) which are responsive on mobile, tablet and desktop
- [x] Very basic routing setup (use Router v6)

**To-do (spare for next PRs, otherwise this PR will have too many changes)**
- [ ] Testing
- [ ] Authentication flow (probably mock token), home page looks like a joke now
- [ ] Anything else?

### CODE REVIEW

File: packages/frontend/src/pages/NotFound/index.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+export const NotFound = () => {
+  return (
+    <Container>
+      <Box component="img" src={RataExtLogo} alt="Logo" />
+      <SubtitleWrapper variant="subtitle2">Sivua ei löydy</SubtitleWrapper>
+      <ParagraphWrapper variant="body1">
+        Hakemasi sivu on väliaikaisesti poissa käytöstä, tai sivua ei ehkä enää ole.
+      </ParagraphWrapper>
+      <ListWrapper>
+        <ListItemText primary="TEE JOKIN SEURAAVUSTA:" />
```
Comment (mehtis) [inline packages/frontend/src/pages/NotFound/index.tsx:16]: SEURAAVUSTA -> SEURAAVISTA

File: packages/frontend/src/pages/NotFound/index.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            primary="
+Tarkista URL-osoitteen oikeinkirjoitus (tarkista isot ja pienet kirjaimet sekä välimerkit) ja napsauta selaimen Päivitä-painiketta."
+          />
+        </ListItemWrapper>
+        <ListItemWrapper sx={{ display: 'list-item' }}>
+          <ListItemText primary="Palaa edelliselle sivulle napsauttamalla selaimen Edellinen-painiketta." />
+        </ListItemWrapper>
+      </ListWrapper>
+      <BackButton href="/" color="primary" variant="contained">
+        Sirry etusivulle
```
Comment (mehtis) [inline packages/frontend/src/pages/NotFound/index.tsx:31]: Sirry -> Siirryy

**Review by mehtis — APPROVED** [review]

File: packages/frontend/src/pages/NotFound/index.tsx (hunk 2 above)
Comment (saneDG) [reply packages/frontend/src/pages/NotFound/index.tsx:31]: *Siirry

Comment (mehtis) [reply packages/frontend/src/pages/NotFound/index.tsx:31]: Nice typo correction correction 😅

**Pull request merged**